block-layer/codecs/dag-pb.md

```diff
@@ -44,13 +44,13 @@ This layout can be expressed with [IPLD Schemas](../../schemas/README.md) as:
 
 ```ipldsch
 type PBLink struct {
-  Hash Bytes
-  Name optional String
-  Tsize optional Int
+  Hash optional Link
+  Name String (implicit "")
+  Tsize Int (implicit "0")
 }
 
 type PBNode struct {
-  Links optional [PBLink]
+  Links [PBLink]
```

Comment:
https://github.com/ipld/go-ipld-prime-proto/blob/38f564004a47f20fe67a4c36a40aec72c0287b2b/coding.go#L37

We _could_ make this optional, but I'm happy with empty-array, it's cleaner IMO